### DISCUSSION TITLE
希望能够提供快速升级版本的途径

### DISCUSSION THREAD

**Discussion by koalaanntree** [kubevela/kubevela#3174] [Ideas] [answered]
目前版本升级比较繁琐，而且容易出错，比如最近从1.2.0升级到1.2.2的时候，基本等于做了删除重装的一系列操作，希望有更加便捷的方式来进行版本的升级，比如页面上的一键升级、脚本的一键升级、提供一键升级命令等方式，另外版本号希望能够在velaUX上做出显示

**Comment by wonderflow** [comment, marked as answer]
Now, you can use `vela install` for this.
